lib/index.js

```diff
@@ -1,15 +1,18 @@
 const { Router } = require('express');
 const { json } = require('body-parser');
+const { Model } = require('objection');
 const Database = require('./db');
 const router = require('./router');
 const Hooks = require('./hooks');
 const Decorators = require('./decorators');
+const activityLogger = require('./activitylog/hooks/logger');
 
 const Case = require('./models/case');
 
 module.exports = (settings = {}) => {
   const db = Database.connect(settings.db);
   Case.db(db);
+  Model.knex(db);
```

Comment:
Don't do this, it breaks everything.

It binds all models globally to this `knex` connection pool. Even models from `@asl/schema` that talk to a completely different database.

You need to bind the individual model class that you're interested in.